v2/README.md

```diff
@@ -26,13 +22,37 @@ For everything to work as expected make sure of the following:
 
 ## Usage
 
-Edit `.env` for your development requirements - you might need to point to local services running in an IDE for example.
+Please follow the instructions in [stacks README](./stacks/README.md) to run each stack
+
+## Code structure
+
+The required configs and scripts have been structured as follows:
+
+### dockerfiles
+
+Contains `Dockerfile.dp-compose` files for services that do not have a `Dockerfile.local` yet. Each repository should have its own `Dockerfile.local`, so this `dockerfiles` folder can be removed when this is the case.
+
+### manifests
+
+Contains docker compose config `yml` files for each service that is required by any of the stacks. These configurations are stack agnostic and define all the necessary env vars to run the services in any possible configuration that might be required by any stack. Each env var has a sensible default value, which will be used if not provided by the stack, and usually corresponds to the default value in the service config.
+
+The files are organised in subfolders according to their type:
+- core-ons: Core services implemented by ONS
+- deps: Dependencies, not implemented by ONS, used by ONS services
+
+### stacks
+
+Contains definitions for each stack, including config overrides and docker compose extension files.
+Each subfolder corresponds to a particular stack and contains at least:
+- {stack}.yml: Extended docker-compose file which uses the manifests for required services.
+  - More information [here](https://docs.docker.com/compose/extends/)
+- .env: With the environmental variables required to override the default config for the services in the stack
+  - More information [here](https://docs.docker.com/compose/environment-variables/#using-the---env-file--option) and [also here](https://docs.docker.com/compose/environment-variables/#using-the---env-file--option)
+
+### provisioning
 
-Then just standard Docker compose commands: e.g.:
-- to start detached: `docker-compose --project-dir . -f profiles/static-files.yml up -d`
-- or with the alias: `dpc up -d`
-- to get logs for a service: `docker-compose --project-dir . -f profiles/static-files.yml logs dp-files-api` or `dpc logs dp-files-api`
+Contains scripts and files to set the initial state required for stacks to work. This include things like database collections, content, etc.
 
 ## Kafka
 
-This uses KRaft'mode but this is early release: https://github.com/apache/kafka/blob/6d1d68617ecd023b787f54aafc24a4232663428d/config/kraft/README.md - have followed this issue but am documenting in case things start to fail: https://github.com/bitnami/bitnami-docker-kafka/issues/159
\ No newline at end of file
+This uses KRaft'mode but this is early release: https://github.com/apache/kafka/blob/6d1d68617ecd023b787f54aafc24a4232663428d/config/kraft/README.md - have followed this issue but am documenting in case things start to fail: https://github.com/bitnami/bitnami-docker-kafka/issues/159
```

Comment:
I don't think there needs to be a link to the issue here, which has recently been archived: https://github.com/bitnami/bitnami-docker-kafka/issues/159. Also there's a single quote to delete in this line:
```suggestion
This uses KRaft mode, which is an early release: https://github.com/apache/kafka/blob/6d1d68617ecd023b787f54aafc24a4232663428d/config/kraft/README.md
```

Reply:
This was from the existing documentation before my changes, I should have reviewed it. I will remove it.

Reply:
👍